packages/server/index.ts

```diff
@@ -0,0 +1,31 @@
+import bodyParser from "body-parser";
+import cors from "cors";
+import dotenv from "dotenv";
+import express from "express";
+import path from "path";
+import { fileURLToPath } from "url";
+
+import rootRouter from "./routes/rootRouter.js";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+dotenv.config({
+  path: path.join(
+    __dirname,
+    process.env.NODE_ENV === "development" ? "../../.env.dev" : "../../.env"
```

Comment:
Similar to above, we're telling the server where to find environment files. These are in the `.gitignore` so will not show up in git, but when we have environment variables, we should make reference to them in an `.env.example` file.